packages/runtime/runtime-utils/src/summarizerNode/summarizerNode.ts

```diff
@@ -228,6 +228,10 @@ export class SummarizerNode implements IRootSummarizerNode {
             const props = {
                 summaryRefSeq,
                 pendingSize: this.pendingSummaries.size ?? undefined,
+                pendingHandle: this.pendingSummaries.size > 0 ?
+                    this.pendingSummaries.keys().next().value : undefined,
```

Comment:
Are you ok with this logging a "random" entry depending on the map hash?

Reply:
I believe so as there will be mostly 1 if ever as we remove previous ones when we skip.